src/transaction/log_manager.c

```diff
@@ -1548,6 +1548,11 @@ log_initialize_internal (THREAD_ENTRY * thread_p, const char *db_fullname, const
 	}
     }
 
+  if (is_tran_server_with_remote_storage () && is_active_transaction_server ())
```

Comment:
If `log_recovery_finish_transactions` is also executed, it also executes a `logpb_checkpoint_trantable`.

I would like us to investigate also a method that involves the daemon.
Implement a custom looper `cubthread::period_function` (see example: `log_get_log_group_commit_interval`) with the following logic:
- when called, as long as `!BO_IS_SERVER_RESTARTED` (this is also the condition used in `log_checkpoint_trantable_execute`), return a wait interval of 1 second - this will cause the daemon thread to call the daemon execute function every second - but it will be useless as the `!BO_IS_SERVER_RESTARTED` condition will also impede the actual creation of the checkpoint
- the first time the condition `BO_IS_SERVER_RESTARTED` is met, also return a wait interval of 1 second to cause the daemon to execute immediately - in this case, the interval can be even smaller
- the rest of the times, return unconditionally an interval of 60 seconds
In order to provide state, use a local static variable in the "period_function" implementation - it is safe to use the static variable as long as only one thread (even if not necessarily the same thread every time) will call it.